packages/cfd/src/Containers/trade-modal.tsx

```diff
@@ -0,0 +1,409 @@
+import React from 'react';
+import { Text, Button, Icon, Money, Popover } from '@deriv/components';
+import { TPasswordBoxProps, TTradingPlatformAccounts, TCFDDashboardContainer } from '../Components/props.types';
+import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+import { CFD_PLATFORMS, getCFDAccountDisplay, getCFDPlatformLabel, getCFDAccountKey, isMobile } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+import { CFDAccountCopy } from '../Components/cfd-account-copy';
+import { QRCode } from 'react-qrcode';
+import {
+    getDXTradeWebTerminalLink,
+    getPlatformDXTradeDownloadLink,
+    getPlatformCTraderDownloadLink,
+    getPlatformDerivEZDownloadLink,
+    getPlatformMt5DownloadLink,
+} from '../Helpers/constants';
+
+type TPlatformType = 'dxtrade' | 'derivez' | 'ctrader';
```

Comment:
Let's define a props.types file for all the containers and move all the types there.

Reply:
Will move it to our props.types.ts file